### PR TITLE
chore: allow test deps to be auto updated; update test dep versions

### LINKED ISSUE #112: Update Rhum to latest
Update Rhum to the latest version so that it doesn't import deno std 0.82.0.

### PR DESCRIPTION
**Pull request by ebebbington**
Fixes #112

### CODE REVIEW

File: test_deps/deps.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { ServerRequest } from "https://deno.land/std@0.92.0/http/server.ts";
+import { ServerRequest } from "https://deno.land/std@0.97.0/http/server.ts";
```
Comment (saragee3) [inline test_deps/deps.ts:1]: why not leave it as `test_deps.ts`?

Comment (ebebbington) [reply test_deps/deps.ts:1]: Limitation of dmm tbh lol, it only updates deps.ts files and no other files with another file name

Comment (crookse) [reply test_deps/deps.ts:1]: sucks, but maybe we could add an issue in dmm to address something like this. or we can just put the test deps.ts file into the tests directory?

Comment (ebebbington) [reply test_deps/deps.ts:1]: > sucks, but maybe we could add an issue in dmm to address something like this. or we can just put the test deps.ts file into the tests directory?

ehhhh sure lol, i mean it makes sense to add this feature to dmm

also there isn't a 'root' tests dir, each middleware dir has its onw tests dir so not really possible

Comment (crookse) [reply test_deps/deps.ts:1]: LMFAO. woops. i totally overlooked that.

**Comment by ebebbington** [comment]
relies on https://github.com/drashland/dmm/pull/114

**Comment by ebebbington** [comment]
closing, this will be addressed once dmm releases and we can update the bumper file

**Pull request closed without merging**